<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VegetablesClass</class>
 <widget class="QMainWindow" name="VegetablesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1716</width>
    <height>1207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vegetables</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="family_list_widget">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="vegetables_list_widget">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="vegetable_label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Vegetable</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="vegetable_line_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="search_button">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="parts_label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Parts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="parts_line_edit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1716</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Vegetables.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>family_list_widget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>VegetablesClass</receiver>
   <slot>select_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>1175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_button</sender>
   <signal>clicked()</signal>
   <receiver>VegetablesClass</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1649</x>
     <y>1117</y>
    </hint>
    <hint type="destinationlabel">
     <x>1598</x>
     <y>1173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_item()</slot>
  <slot>search()</slot>
 </slots>
</ui>
